osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/config/SoapClientConfig.java

```diff
@@ -82,11 +82,11 @@ public HttpsUrlConnectionMessageSender httpsUrlConnectionMessageSender() throws
         HttpsUrlConnectionMessageSender httpsUrlConnectionMessageSender =
                 new HttpsUrlConnectionMessageSender();
         // set the trust store(s)
-        httpsUrlConnectionMessageSender.setTrustManagers(trustManagersFactoryBean().getObject());
+        httpsUrlConnectionMessageSender.setTrustManagers(this.trustManagersFactoryBean().getObject());
         // set the key store(s)
-        httpsUrlConnectionMessageSender.setKeyManagers(keyManagersFactoryBean().getObject());
+        httpsUrlConnectionMessageSender.setKeyManagers(this.keyManagersFactoryBean().getObject());
 
-        if (Boolean.parseBoolean(useHostNameVerifier) == false) {
+        if (Boolean.parseBoolean(this.useHostNameVerifier) == false) {
             httpsUrlConnectionMessageSender.setHostnameVerifier(new NoopHostnameVerifier());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1125)